examples/README.md

```diff
@@ -0,0 +1,12 @@
+# go-tpm usage examples
+
+This directory contains binaries demonstrating usage of go-tpm.
+
+## Versions
+
+All directories that start with `tpm-` are for TPM 1.x devices.
```

Comment:
[optional]: another simplification: drop the "All" from each sentence:

"Directories that start with "...

Reply:
Done